ecosystem_tests/ecosystem_tests_cli/main.py

```diff
@@ -14,7 +14,8 @@
 # limitations under the License.
 
 from ..ecosystem_tests_cli import ecosystem_tests
-from .commands import (validate_blueprint,
+from .commands import (validate_plugin_version,
+                       validate_blueprint,
                        local_blueprint_test,
                        prepare_test_manager)
 
```

Comment:
Same here, need to move `validate_plugin_version` to be after `prepare_test_manager`, because Cloudify code puts imports in order of name length.